google-userinfo/background/authorize.js

```diff
@@ -0,0 +1,74 @@
+const REDIRECT_URL = browser.identity.getRedirectURL();
+const CLIENT_ID = "YOUR-CLIENT-ID";
+const SCOPES = ["openid", "email", "profile"];
+const AUTH_URL =
```

Comment:
Do you mean I should indent the pieces of AUTH_URL, like:

```
const AUTH_URL = `https://accounts.google.com/o/oauth2/auth
  ?client_id=${CLIENT_ID}
  &response_type=token
  &redirect_uri=${encodeURIComponent(REDIRECT_URL)}
  &scope=${encodeURIComponent(SCOPES.join(' '))}`;
```
? I didn't do this because then you get spaces in the URL. Sorry if I'm being slow-witted here though.